### PR TITLE
KO: Translations for ko/beta/tutorials/eager

### PR DESCRIPTION
**Pull request by Choiuijin1125**
This PR is re-uploaded from 633PR(https://github.com/tensorflow/docs/pull/633) because of commit issue.

### CODE REVIEW

**Comment by rickiepark** [comment]
@Choiuijin1125 Please move the files to beta/tutoials/eager. :)

**Comment by Choiuijin1125** [comment]
@rickiepark Ohhh, thanks :)

File: site/ko/beta/tutorials/eager/automatic_differentiation.ipynb [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "1CLWJl0QliB0"
+   },
+   "source": [
+    "## 그래디언트 테이프\n",
+    "\n",
+    "텐서플로는 자동 미분(주어진 입력 변수에 대한 연산의 그래디언트(gradient)를 계산하는 것)을 위한 [tf.GradientTape](https://www.tensorflow.org/api_docs/python/tf/GradientTape) API를 제공합니다. `tf.GradientTape`는 안에서 실행된 모든 연산을 테이프(tape)에 \"기록\"합니다. 그리고 [후진 방식 자동 미분(reverse mode differentiation)](https://en.wikipedia.org/wiki/Automatic_differentiation)을 사용한 \"기록된\" 계산의 그래디언트를 계산하기 위해 기록된 연산에 관련된 테이프와 그래디언트를 사용합니다.\n",
```
Comment (rickiepark) [inline site/ko/beta/tutorials/eager/automatic_differentiation.ipynb]: tf.GradientTape's context is meaningful to most python programmers.
I suggest "tf.GradientTape는 안에서" --> "tf.GradientTape 컨텍스트(context) 안에서".

Most talented Korean writers advise not too much to use conjunctions.(그리고, 그래서, 하지만.. etc) But I think that you can translate 'then'.
Last sentence has duplicate context like 'TF use gradient to compute gradient'. It can be simplifed just like 'TF compute gradient'.
I suggest
'그리고 후진 방식 자동 미분(reverse mode differentiation)을 사용한 "기록된" 계산의 그래디언트를 계산하기 위해 기록된 연산에 관련된 테이프와 그래디언트를 사용합니다.'
-->
'그다음 텐서플로는 후진 방식 자동 미분(reverse mode differentiation)을 사용해 테이프에 "기록된" 연산의 그래디언트를 계산합니다.'

File: site/ko/beta/tutorials/eager/automatic_differentiation.ipynb [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "N4VlqKFzzGaC"
+   },
+   "source": [
+    "또한 `tf.GradientTape` 컨텍스트 안에서 기록되는 동안 계산된 중간값에 대한 그래디언트도 구할 수 있습니다."
```
Comment (rickiepark) [inline site/ko/beta/tutorials/eager/automatic_differentiation.ipynb]: Just remove '기록되는 동안' and read again. It's much better. :)

File: site/ko/beta/tutorials/eager/automatic_differentiation.ipynb [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "4U1KKzUpNl58"
+   },
+   "source": [
+    "## 다음 단계\n",
+    "\n",
+    "이번 튜토리얼에서는 텐서플로에서 그래디언트 계산법을 다루었습니다. 이를 통해 신경망(neural network)을 구축하고 훈련시키는데 필요한 많은 기본 요소를 배웠습니다."
```
Comment (rickiepark) [inline site/ko/beta/tutorials/eager/automatic_differentiation.ipynb]: You can choose "텐서플로에서 그래디언트를 계산하는 방법을" or "텐서플로의 그래디언트 계산법을". :)

File: site/ko/beta/tutorials/eager/automatic_differentiation.ipynb [hunk 4]
```diff
@@ -0,0 +1,351 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "DK05KXrAAld3"
+   },
+   "source": [
+    "### 고계도 그래디언트\n",
+    "\n",
+    "`GradientTape` 컨텍스트 매니저안에 있는 연산들은 자동미분을 위해 기록됩니다. 만약 그래디언트가 컨텍스트 안에서 계산되면 해당 그래디언트 연산 또한 기록되어집니다. 그 결과 똑같은 API가 고계도(Higher-order) 그래디언트에서도 잘 작동합니다. 예를 들면:"
```
Comment (rickiepark) [inline site/ko/beta/tutorials/eager/automatic_differentiation.ipynb]: To make it more readable,
I suggest '그래디언트가 컨텍스트 안에서 계산되면' --> '이 컨텍스트 안에서 그래디언트를 계산하면'

File: site/ko/beta/tutorials/eager/basics.ipynb [hunk 5]
```diff
@@ -0,0 +1,490 @@
...
+   "source": [
+    "### 넘파이 호환성\n",
+    "\n",
+    "텐서와 넘파이 배열 사이의 변환은 다소 간단합니다.\n",
+    "\n",
+    "* 텐서플로 연산은 자동적으로 넘파이 배열을 텐서로 변환합니다.\n",
+    "* 넘파이 연산은 자동적으로 텐서를 넘파이 배열로 변환합니다.\n",
+    "\n",
+    "텐서는 `.numpy()` 메서드(method)를 호출하여 넘파이 배열로 변환할 수 있습니다.\n",
+    "가능한 경우, `tf.Tensor`와 배열은 메모리 표현을 공유하기 때문에 이러한 변환은 일반적으로 간단(저렴)합니다. 그러나 `tf.Tensor`는 GPU 메모리에 저장될 수 있고, 넘파이 배열은 항상 호스트 메모리에 저장되므로, 이러한 변환이 항상 가능한 것은 아닙니다. 따라서 GPU에서 호스트 메모리로의 복사가 필요합니다."
```
Comment (rickiepark) [inline site/ko/beta/tutorials/eager/basics.ipynb]: For the same reason, please don't use '의' unnecessarily. We are familiar with '~의' (e.g '나에게로의 초대'). But it's from translating Japanese(の) as it is. 
I suggest '메모리로의 복사가' --> '메모리로 복사가'

File: site/ko/beta/tutorials/eager/basics.ipynb [hunk 6]
```diff
@@ -0,0 +1,490 @@
...
+    "colab_type": "code",
+    "id": "lCUWzso6mbqR"
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "ndarray = np.ones([3, 3])\n",
+    "\n",
+    "print(\"텐서플로 연산은 자동적으로 넘파이 배열을 텐서로 변환합니다.\")\n",
```
Comment (rickiepark) [inline site/ko/beta/tutorials/eager/basics.ipynb]: '자동적으로' --> '자동으로'

File: site/ko/beta/tutorials/eager/basics.ipynb [hunk 7]
```diff
@@ -0,0 +1,490 @@
...
+    "tensor = tf.multiply(ndarray, 42)\n",
+    "print(tensor)\n",
+    "\n",
+    "\n",
+    "print(\"그리고 넘파이 연산은 자동적으로 텐서를 넘파이 배열로 변환합니다.\")\n",
```
Comment (rickiepark) [inline site/ko/beta/tutorials/eager/basics.ipynb]: '자동적으로' --> '자동으로'

File: site/ko/beta/tutorials/eager/basics.ipynb [hunk 8]
```diff
@@ -0,0 +1,490 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "PBNP8yTRfu_X"
+   },
+   "source": [
+    "## GPU 가속\n",
+    "\n",
+    "대부분의 텐서플로 연산은 GPU를 사용하여 가속화됩니다. 어떠한 코드를 명시하지 않아도, 텐서플로는 필요시 텐서를 CPU 와 GPU에 연산을 복사하기 위해 자동적으로 CPU 또는 GPU를 사용할 것인지를 정합니다. 연산에 의해 생성된 텐서는 전형적으로 연산이 실행된 장치의 메모리에 의해 실행됩니다. 예를 들어:"
```
Comment (rickiepark) [inline site/ko/beta/tutorials/eager/basics.ipynb]: To consider 'automatically decides', it's better to move '자동적으로'.
It's better to split the sentence at 'copying ~'.
I suggest 
'텐서플로는 필요시 텐서를 CPU 와 GPU에 연산을 복사하기 위해 자동적으로 CPU 또는 GPU를 사용할 것인지를 정합니다'
-->
'텐서플로는 연산을 위해 CPU 또는 GPU를 사용할 것인지를 자동으로 결정합니다. 필요시 텐서를 CPU와 GPU 메모리 사이에서 복사합니다'

File: site/ko/beta/tutorials/eager/basics.ipynb [hunk 9]
```diff
@@ -0,0 +1,490 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "A8X1GNfoIZKJ"
+   },
+   "source": [
+    "### 반복\n",
+    "`tf.data.Dataset`은 레코드 순회를 지원하는 반복가능한 객체입니다.\""
```
Comment (rickiepark) [inline site/ko/beta/tutorials/eager/basics.ipynb]: Just remove \" at the end. :)

File: site/ko/beta/tutorials/eager/custom_layers.ipynb [hunk 10]
```diff
@@ -0,0 +1,406 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "tk8E2vY0-z4Z"
+   },
+   "source": [
+    "다른 독자가 표준형 층의 동작을 잘 알고 있기 때문에, 가능한 경우 표준형 층을 사용하는것이 전체 코드를 읽고 유지하는데 더 쉽습니다. 만약 `tf.keras.layers` 에 없는 층을 사용하기 원하면 [깃허브](http://github.com/tensorflow/tensorflow/issues/new)에 이슈화하거나, 풀 리퀘스트(pull request)를 보내세요."
```
Comment (rickiepark) [inline site/ko/beta/tutorials/eager/custom_layers.ipynb]: It's naive to translate 'reader' as '독자'. :)
To make it cool, I suggest '다른 독자가 표준형 층의 동작을 잘 알고 있기 때문에' --> '코드를 읽는 사람이 표준형 층의 동작을 잘 알고 있을 것이므로'

File: site/ko/beta/tutorials/eager/custom_layers.ipynb [hunk 11]
```diff
@@ -0,0 +1,406 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "Qhg4KlbKrs3G"
+   },
+   "source": [
+    "## 모델: 층 구성\n",
+    "\n",
+    "머신러닝 모델에서 대부분의 재미있는 많은 것들은 기존의 층을 조합하여 구현됩니다. 예를 들어, 레스넷(resnet)의 각 잔여 블록(residual block)은 합성곱(convolution), 배치 정규화(batch normalization), 쇼트컷(shortcut) 등으로 구성되어 있습니다. \n",
```
Comment (rickiepark) [inline site/ko/beta/tutorials/eager/custom_layers.ipynb]: Most people in this field say resnet as '레즈넷' rather than '레스넷'.
I suggest '레스넷' --> '레즈넷'

File: site/ko/beta/tutorials/eager/custom_layers.ipynb [hunk 12]
```diff
@@ -0,0 +1,406 @@
...
+    "\n",
+    "다른층을 포함한 모델을 만들기 위해 사용하는 메인 클래스는 tf.keras.Model입니다. 다음은 tf.keras.Model을 상속(inheritance)하여 구현한 코드입니다."
```
Comment (rickiepark) [inline site/ko/beta/tutorials/eager/custom_layers.ipynb]: Just add a space. '다른층을' --> '다른 층을'

File: site/ko/beta/tutorials/eager/custom_layers.ipynb [hunk 13]
```diff
@@ -0,0 +1,406 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "wYfucVw65PMj"
+   },
+   "source": [
+    "그러나 대부분의 경우에, 많은 층으로 구성된 모델은 간단하게 연이어 하나의 층으로 호출할 수 있습니다. 이는 tf.keras.Sequential 사용하여 간단한 코드로 구현 가능합니다."
```
Comment (rickiepark) [inline site/ko/beta/tutorials/eager/custom_layers.ipynb]: 'one layer after the other' means 'one by one'.
'하나의 층' is slightly incorrect.
I suggest '간단하게 연이어 하나의 층으로 호출할 수 있습니다' --> '단순하게 순서대로 층을 하나씩 호출합니다'

File: site/ko/beta/tutorials/eager/custom_training.ipynb [hunk 14]
```diff
@@ -0,0 +1,467 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "BMiFcDzE7Qu3"
+   },
+   "source": [
+    "## 예: 선형 모델 훈련\n",
+    "\n",
+    "지금까지 몇 가지 개념을 설명했습니다. 간단한 모델을 구축하고 학습시키기 위해 ---`Tensor`, `GradientTape`, `Variable` --- 등을 사용하였고, 이는 일반적으로 다음의 과정을 포함합니다.\n",
```
Comment (rickiepark) [inline site/ko/beta/tutorials/eager/custom_training.ipynb]: You may split the sentence incorrectly. :)
I suggest
'지금까지 몇 가지 개념을 설명했습니다. 간단한 모델을 구축하고 학습시키기 위해 ---Tensor, GradientTape, Variable --- 등을 사용하였고,'
-->
'지금까지 간단한 모델을 구축하고 학습시키기 위해 Tensor, GradientTape, Variable와 같은 몇 가지 개념을 설명했습니다.'

File: site/ko/beta/tutorials/eager/custom_training.ipynb [hunk 15]
```diff
@@ -0,0 +1,467 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "sSDP-yeq_4jE"
+   },
+   "source": [
+    "### 훈련 루프 정의\n",
+    "\n",
+    "이제 네트워크와 훈련 데이터가 준비되었습니다. 모델의 변수(`W` 와 `b`)를 업데이트하기 위해 훈련 데이터를 사용하여 훈련시켜 보죠. 그리고 [경사 하강법(gradient descent)](https://en.wikipedia.org/wiki/Gradient_descent)을 사용하여 손실을 감소시킵니다. 경사 하강법에는 여러가지 방법이 있으며, `tf.train.Optimizer` 에 구현되어있습니다. 이러한 구현을 사용하는것을 강력히 추천드립니다. 그러나 이번 튜토리얼에서는 기본적인 방법을 사용하겠습니다."
```
Comment (rickiepark) [inline site/ko/beta/tutorials/eager/custom_training.ipynb]: Just add a space. '사용하는것을' --> '사용하는 것을'

File: site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb [hunk 16]
```diff
@@ -0,0 +1,1159 @@
...
+    "\n",
+    "<table>\n",
+    "  <tr><td>\n",
+    "    <img src=\"https://www.tensorflow.org/images/custom_estimators/full_network.png\"\n",
+    "         alt=\"A diagram of the network architecture: Inputs, 2 hidden layers, and outputs\">\n",
+    "  </td></tr>\n",
+    "  <tr><td align=\"center\">\n",
+    "    <b>그림 2.</b> A neural network with features, hidden layers, and predictions.<br/>&nbsp;\n",
```
Comment (rickiepark) [inline site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb]: Please translate the title of 그림 2.

File: site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb [hunk 17]
```diff
@@ -0,0 +1,1159 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "FHcbEzMpxbHL"
+   },
+   "source": [
+    "*[사용가능한 활성화 함수(activation function)](https://developers.google.com/machine-learning/crash-course/glossary#activation_function)*는 각 층에서 출력의 크기를 결정합니다. 이러한 비선형성은 중요하며, 활성화 함수가 없는 모델은 하나의 층과 동일하다고 생각할 수 있습니다. 사용 가능한 [활성화 함수](https://www.tensorflow.org/api_docs/python/tf/keras/activations)는 많지만, [ReLU](https://developers.google.com/machine-learning/crash-course/glossary#ReLU)가 은닉층에 주로 사용됩니다. \n",
```
Comment (rickiepark) [inline site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb]: Shape is often translated '크기'. But I think 'shape' in this sentence means '형태'. Because activation func. doesn't change Tensor shape. And just remove '사용가능한'.
I suggest '사용가능한 활성화 함수(activation function)는 각 층에서 출력의 크기를 결정합니다' --> '활성화 함수(activation function)는 각 층에서 출력의 형태를 결정합니다'

File: site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb [hunk 18]
```diff
@@ -0,0 +1,1159 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "pJVRZ0hP52ZB"
+   },
+   "source": [
+    "이 값들을 단일 최적화 단계를 계산하기 위해 사용합니다. "
```
Comment (rickiepark) [inline site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb]: You can translate 'use' first. :)
I suggest '이 값들을 단일 최적화 단계를 계산하기 위해 사용합니다' --> '이를 사용해 한 번의 최적화 단계를 계산합니다'

File: site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb [hunk 19]
```diff
@@ -0,0 +1,1159 @@
...
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "agjvNd2iUGFn"
+   },
+   "outputs": [],
+   "source": [
+    "fig, axes = plt.subplots(2, sharex=True, figsize=(12, 8))\n",
+    "fig.suptitle('훈련 메트릭스')\n",
```
Comment (rickiepark) [inline site/ko/beta/tutorials/eager/custom_training_walkthrough.ipynb]: metric can be translated '지표' or '측정 지표'.

**Comment by Choiuijin1125** [comment]
Thank you for the great reviews. :) 
@rickiepark, @cre8to,r Can you guys check this PR again? :)
I checked Colab also. Thanks!

**Comment by Choiuijin1125** [comment]
Hi, @lamberta Can you check this PR? :)
